docs/specification/current.md

```diff
@@ -328,7 +328,7 @@ goal is to know, at call time:
 | --- | ----------- |
 | `patient` | Patient demographics. |
 | `hemoglobin-a1c` | Most recent Hemoglobin A1c reading for this patient. |
-| `user` | Information on the current user (Practitioner).
+| `userId` | Information on the current user (PractitionerRole).
```

Comment:
This 'user' prefetch key is referring to resolving the userId reference and doesn't necessitate renaming to 'userId'.

Reply:
reverted. Thanks, Dennis.